Sources/EZSwiftExtensions.swift

```diff
@@ -11,29 +11,29 @@ import UIKit
 
 public struct ez {
 
-    /// EZSwift Extensions
+    /// EZSE: Returns app's name
     public static var appDisplayName: String {
         return NSBundle.mainBundle().objectForInfoDictionaryKey("CFBundleDisplayName") as? String
             ?? NSBundle.mainBundle().objectForInfoDictionaryKey("CFBundleName") as! String
     }
 
-    /// EZSwift Extensions
+    /// EZSE: Returns app's version number
     public static var appVersion: String {
         return NSBundle.mainBundle().objectForInfoDictionaryKey("CFBundleShortVersionString") as! String
     }
 
-    /// EZSwiftExtensions
+    /// EZSE: Return app's build number
     public static var appBuild: String {
         return NSBundle.mainBundle().objectForInfoDictionaryKey(kCFBundleVersionKey as String) as! String
     }
 
-    /// EZSwiftExtensions
+    /// EZSE: Returns both app's version and build numbers
     public static var appVersionAndBuild: String {
```

Comment:
Return format is not clear from the documentation.The format of the return value could be documented as well.